lib/jobs/uefi-job.js

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+
+module.exports = uefiJobFactory;
+di.annotate(uefiJobFactory, new di.Provide('Job.Uefi'));
```

Comment:
I have some concern about the job's name, the `Job.Uefi` is a so general word, seems it will cover all UEFI related work, but as you described in the PR, this is used to run UEFI application, so `Job.Uefi.App` or `Job.RunUefiApp` seems better.